src/status_im/utils/datetime.cljs

```diff
@@ -46,7 +47,7 @@
 
 (defn- to-str [ms old-fmt-fn yesterday-fmt-fn today-fmt-fn]
   (let [date (from-long ms)
-        local (plus date time-zone-offset)
+        local (plus date time-zone-offset) ; this is wrong, it uses the current timezone offset, regardless of DST
```

Comment:
Maybe create a separate issue around that problem?

Reply:
Yes will do